README.md

```diff
@@ -132,21 +130,22 @@ curl --header "Content-Type: application/json" \
 
 ### Checking the faucets status
 
-The faucet provides a simple status check in the form of an http GET request. As above, make sure to adjust the URL as necessary.
+The faucet provides a simple status check in the form of an http GET request. As
+above, make sure to adjust the URL as necessary.
 
 ```
 curl http://localhost:8000/status
 ```
 
 ## Versions and compatibility overview
 
-| iov-faucet | IOV-Core | BNSd support    | New features     |
-| ---------- | -------- | --------------- | ---------------- |
-| 0.10.x     | 1.2.x    | 0.22.x          |                  |
-| 0.9.x      | 0.17.x   | 0.21.x          |                  |
-| 0.8.x      | 0.16.x   | 0.19.x – 0.20.x |                  |
-| 0.7.x      | 0.15.x   | 0.16.x          |                  |
-| 0.6.x      | 0.14.x   | 0.14.x          | BNS fee support  |
-| 0.5.x      | 0.12.x   | 0.10.x – 0.11.x |                  |
-| 0.4.x      | 0.11.x   | 0.4.x – 0.9.x   | Ethereum support |
-| 0.3.x      | 0.9.x    | 0.4.x – 0.9.x   |                  |
+| iov-faucet | IOV-Core | BNSd support    | New features                                                              |
```

Comment:
Was this table formatted by prettier?

Reply:
Yes it was, by running `yarn format-text`

Reply:
It doesn't look like the columns all have the same length for each row.

Reply:
I little bit hard to review because of the long lines but they do: https://raw.githubusercontent.com/iov-one/iov-faucet/simplify-path/README.md